package.json

```diff
@@ -41,7 +42,8 @@
     "@babel/preset-env": "^7.11.5",
     "@babel/preset-typescript": "^7.10.4",
     "@preconstruct/cli": "^1.1.27",
-    "react": "^16.13.1"
+    "react": "^16.13.1",
+    "vue": "3.0.0-rc.10"
```

Comment:
Vue 3.0 is released already https://github.com/vuejs/vue-next/releases

Reply:
Good spot buddy. I'll update this when I get some free time :)